koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -444,22 +436,26 @@ def createContainer(self, src=None, target_info=None, arch=None, arches=None,
             'scratch': scratch,
             'koji_task_id': self.id,
         }
-        if arches:
-            create_build_args['platforms'] = arches
-            create_method = self.osbs().create_orchestrator_build
-        elif arch:
-            create_build_args['platform'] = arch
-            create_method = self.osbs().create_build
-        else:
-            raise ContainerError("Neither arch nor arches specified")
+        if not arches:
+            raise ContainerError("arches aren't specified")
 
         if branch:
             create_build_args['git_branch'] = branch
         if push_url:
             create_build_args['git_push_url'] = push_url
 
         self.logger.debug("Starting %s with params: '%s'", create_method, create_build_args)
-        build_response = create_method(**create_build_args)
+
+        try:
+            self.logger.debug("Starting create_orchestrator_build with params: '%s, platforms=%r'",
+                              create_build_args, arches)
+            build_response = self.osbs().create_orchestrator_build(**create_build_args, platforms=arches)
```

Comment:
`create_build_args` and `create_build_method` is probably a cleaner way to do this